### PR TITLE
Miscellaneous fixes for new script

### PR DESCRIPTION
**Pull request by pradh**
(1) Support fulfillment of ACROSS contained-in type by resolving to a place-type based on the place considered
(2) Support fulfillment of EXTREME ranking type by deciding whether to show lowest based on SV name
(3) Prune some SV names per Guha request (a hack for now)
(4) Modify the RANKING chart display to always show an adjoining map on the same row
(5) Drop a weird jobs SV from topic
(6) Make and use helper functions for getting {ranking-type, contained-in, time-delta-types} from utterance
(7) Fix the temperature hack properly to avoid pinned max dates

Sorry that (6) is in this PR.  I have tested both the old and new scripts with this though.

For the main UI change of ranking-across-places results, here is an example screenshot.

Screenshot 2023-02-03 at 10 06 25 PM

### CODE REVIEW

**Comment by rvguha** [comment]
approve

**Review by rvguha — APPROVED** [review]
looks good

**Comment by pradh** [comment]
@beets @shifucun - please have a look when you find time.  submitting to review script on autopush.

File: server/lib/nl/utils.py [hunk 1]
```diff
@@ -371,13 +373,34 @@ def get_sv_name(all_svs: List[str]) -> Dict:
...
+      sv_name_map[sv] = clean_sv_name(title_by_sv_dcid[sv])
     else:
-      sv_name_map[sv] = uncurated_names[sv]
+      sv_name_map[sv] = clean_sv_name(uncurated_names[sv])
 
   return sv_name_map
 
 
+# TODO: Remove this hack by fixing the name in schema and config.
+def clean_sv_name(name: str) -> str:
```
Comment (beets) [inline server/lib/nl/utils.py:384]: there's a similar one in https://github.com/datacommonsorg/website/blob/master/server/lib/nl/descriptions.py#L37

do you also want to add "Percentage of"?

**Review by beets — COMMENTED** [review]
nice, just a small comment

**Pull request merged**